src/ducks/export/helpers.js

```diff
@@ -20,6 +20,7 @@ export const isExportJobInProgress = async client => {
       .partialIndex({
         $or: [{ state: 'queued' }, { state: 'running' }]
       })
+      .indexFields(['worker', 'message.slug', 'message.name', 'state'])
```

Comment:
`state` est dans le `partialIndex` et non dans `where`, j'avais noté que ce n'était donc pas nécessaire de le mettre dans les `indexFields` ? @paultranvan

Reply:
il n'y a pas de `.sortBy` ici... 🤔 si ?

Reply:
Cf 1 ligne en-dessous :laughing:

Reply:
@taratatach Le point important ici est que `.indexFields(['worker'])` est suffisant, pas besoin d'indexer le reste